src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx

```diff
@@ -148,6 +152,56 @@ Dashboards features include:
 
     You can edit the name and other properties of the dashboard, such as permissions, any time.
   </Collapser>
+
+<Collapser
+    id="dash-color"
+    title="Color customization"
+>
+    You can customize the color of your charts for a specific series in area, bar, line, pie, and stacked bar chats.
+  
+   To customize the color in a serie:
```

Comment:
```suggestion
   To customize the color in a series:
```

Reply:
I am uncertain if "serie" is an industry term, or if it should be series whenever it says serie. But calling it out here just in case. :)

Reply:
Thank you @akristen! I've modified the text and now it's: "To customize the color per series" as it's said in the UI